.github/workflows/release-note-label.yml

```diff
@@ -0,0 +1,62 @@
+# A workflow to validate only one release-note label is selected
+
+name: Release-note label
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
```

Comment:
Add `edited` to support the following scenario:

- A user pushes a commit but no release-note label is selected -> check fails
- The user manually select one of release-note labels in the PR description -> check succeeds (without pushing another commit)

Reply:
Without `edited`, we'd need to send another commit to trigger the check, which is obviously inconvenient.